lib/lograge/log_subscribers/base.rb

```diff
@@ -35,11 +37,29 @@ def extract_request(event, payload)
         data.deep_merge!(custom_options(event))
       end
 
-      %i[initial_data extract_error extract_status extract_runtimes
+      %i[initial_data datadog_trace extract_error extract_status extract_runtimes
          extract_location extract_unpermitted_params].each do |method_name|
         define_method(method_name) { |*_arg| {} }
       end
 
+      def datadog_trace
```

Comment:
I think it would be useful to link to the docs here, so that we know where this configuration came from:
```suggestion
      def datadog_trace
      # Manually inject trace IDs,
      # see https://docs.datadoghq.com/tracing/connect_logs_and_traces/ruby/#manual-injection
```

Reply:
Thanks, will add.